yarn.lock

```diff
@@ -2902,14 +2902,10 @@ extract-text-webpack-plugin@^3.0.0:
     schema-utils "^0.3.0"
     webpack-sources "^1.0.1"
 
-extsprintf@1.3.0:
+extsprintf@1.3.0, extsprintf@^1.2.0:
```

Comment:
Are these `yarn.lock` changes necessary? Can we just remove them from this PR if not? I'm not sure what caused this, but make sure `yarn --version` is at least >= 1.x.x.

Reply:
I had a yarn version < 1.x.x, it seems that was the issue.